src/providers/object-provider.js

```diff
@@ -407,6 +410,18 @@ export default class YamcsObjectProvider {
                 this.roleStatusTelemetry.setTelemetryObjectForRole(role, obj);
             }
 
+            if (isMissionStatusParameter(parameter)) {
```

Comment:
Since this is a draft, this is probably already on your radar, but if you're adding a new telemetry parameter, you'll need to add a new type for it for Open MCT so that you visualize the telemetry there. here: https://github.com/akhenry/openmct-yamcs/pull/415/files#diff-44bc2277c0e7dc0a2f4cf8738a83bb06ab31e2abc0a762c0c5f97ded966187e0R200

Reply:
ah, good catch. Thanks